linera-core/src/client/mod.rs

```diff
@@ -2987,29 +2987,11 @@ where
         &self,
         committee: Committee,
     ) -> Result<ClientOutcome<ConfirmedBlockCertificate>, ChainClientError> {
-        loop {
```

Comment:
We're removing the loop here so it's not only a simplification.

Reply:
Ahh, I see the same is done inside `execute_operation`.

Reply:
Similarly to the other discussion - we're trying to achieve the same thing using different layers of abstractions and that's a sign of bad separation of concerns IME.

Reply:
This loop was probably meant to work around https://github.com/linera-io/linera-protocol/issues/3172, which needs a proper solution. (See PR description.)

It's not really useful in this particular case since the admin chain is single-owner anyway.